ext/sqlite3/sqlite3.stub.php

```diff
@@ -86,10 +86,10 @@ class SQLite3Stmt
     private function __construct(SQLite3 $sqlite3, string $sql) {}
 
     /** @return bool */
-    public function bindParam(string|int $param_number, mixed &$param, int $type = SQLITE3_TEXT) {}
+    public function bindParam(string|int $param, mixed &$bind_var, int $type = SQLITE3_TEXT) {}
```

Comment:
```suggestion
    public function bindParam(string|int $index, mixed &$bind_var, int $type = SQLITE3_TEXT) {}
```

Consistency with PDO, and because as $param I kept thinking to put a string there, not an index number.

Reply:
That is ok, but `$index` could confuse the other way around. This can be either positional param or named param. `$param` makes the most sense in my opinion because the function is called `bindParam()` and you are binding a variable to the param. We could also call it something different entirely e.g. `$placeholder`, but I like this even less. 
I'd stick with `$param` for both SQLlite3 and PDO.

Reply:
PDO was actually already using `$param` in this context.